pkg/api/types.go

```diff
@@ -1343,6 +1345,8 @@ func (p ClusterProfile) ClusterType() string {
 		return string(CloudGCP)
 	case ClusterProfileIBMCloud:
 		return "ibmcloud"
+	case ClusterProfileIBMCloudQE:
+		return "ibmcloud-qe"
```

Comment:
Here it is returning `ClusterProfile`, so the value should be "ibmcloud", and this case should be merged to 1346 line.